taskcluster/docker/base-android/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# %ARG DOCKER_IMAGE_PARENT
+FROM $DOCKER_IMAGE_PARENT
+
+MAINTAINER Johan Lorenzo <jlorenzo+tc@mozilla.com>
+
+RUN apt-get install -y openjdk-8-jdk \
+    && apt-get clean
+
+USER worker:worker
+
+COPY install-sdk.sh ./
+
+RUN "$SHELL" -c "source install-sdk.sh"
```

Comment:
Same rationale: Sourcing this script allows ANDROID_SDK_ROOT to be used by gradle in taskgraph. Calling `/bin/bash -c "source foo.sh"` is the only way to import a shell script in Docker, [per stackoverflow](https://stackoverflow.com/questions/20635472/using-the-run-instruction-in-a-dockerfile-with-source-does-not-work).